convert2rhel/main.py

```diff
@@ -298,3 +302,24 @@ def rollback_changes():
             raise
 
     return
+
+
+def print_data_collection():
```

Comment:
I'd say this would be better suited for the breadcrumbs.py module. We should strive for having main module short, serving more as recipe where we gather ingredients from other sources.